pkg/streamingpromql/testdata/ours/name_label_dropping.test

```diff
@@ -95,8 +95,10 @@ eval instant at 10m sum by (__name__, foo) (down or rate(up[5m]))
   down{foo="bar2"} 40
   {foo="bar3"} 0.08333333333333334
 
-eval instant at 10m sum by (__name__) (up{foo="bar"} or rate(up{foo="bar2"}[5m]))
-  up 10.05
+# Temporarily comment out flaky tests.
```

Comment:
Could you please add a link to https://github.com/prometheus/prometheus/pull/17161 here?